src/page/GroupComplete.tsx

```diff
@@ -0,0 +1,90 @@
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+import { Thumbnail } from '../assets';
+import CommonButton from '../common/CommonButton';
+import PageLayout from '../components/PageLayout/PageLayout';
+import { GroupCompleteProps } from '../types/GroupComplte/CompleteType';
+
+const GroupComplete = ({ thumbnailUrl, groupPassword }: GroupCompleteProps) => {
+  const navigate = useNavigate();
+  const baseUrl = window.location.origin; // 생성한 그룹 페이지가 만들어지면 대체 될 예정
+
+  const handleCopyClipBoard = async (text: string) => {
+    try {
+      await navigator.clipboard.writeText(text);
+      alert('클립보드에 링크가 복사되었어요.');
+    } catch (err) {
+      console.log(err);
+    }
+  };
+
+  const handleGroupPageRedirect = () => {
+    alert('그룹 페이지를 생성하고 유지보수 예정.');
+    navigate('/group-page'); // 그룹 페이지 생성 후 유지보수 예정
+  };
+
+  return (
+    <PageLayout category={'group_create'}>
+      <Title>그룹 생성이 완료되었어요!</Title>
+      <PasswordContainer>
+        <PasswordText>
+          비밀번호 <Password>{groupPassword}</Password>
+        </PasswordText>
+      </PasswordContainer>
+      <ThumbnailContainer>
+        {thumbnailUrl ? (
+          <img src={thumbnailUrl} alt="사용자 썸네일" />
+        ) : (
+          <Thumbnail />
+        )}
+      </ThumbnailContainer>
```

Comment:
P1: 이미지 첨부는 그룹 생성과정에서 필수요소이기 때문에, `<Thumbnail />` 컴포넌트를 반환하는 경우는 없습니다!
분기처리 없이, 아래처럼 구현해주시면 될 것 같아용
```suggestion
          <img src={thumbnailUrl} alt="사용자 썸네일" />
```

++ 피그마보시구 이미지 크기 넣어주시면 좋겟네용

Reply:
네엡 감사합니다 !

Reply:
ASK : 그룹생성 컴포넌트에서 이미지를 받아오면 GroupComplete 에서 이미지를 띄워줘야 할텐데 분기처리가 되어야 하지 않나요?

Reply:
이미지는 항상 받아오는 요소이니까 분기처리가 아니라 이미지를 띄워주는 위의 코드만 있으면 되지 않나용?!

Reply:
> 이미지는 항상 받아오는 요소이니까 분기처리가 아니라 이미지를 띄워주는 위의 코드만 있으면 되지 않나용?!

아 항상이여서 그냥 ImageUrl 만 필요하다는 말이죠?!

Reply:
넵 !